### PR TITLE
Create nft与数字身份随想.md

### PR DESCRIPTION
**Pull request by nigdaemon**
xiakezhang PR

### CODE REVIEW

**Review by bobjiang — APPROVED** [review]
Like this idea

**Review by jhfnetboy — DISMISSED** [review]
Great~
It seems you should hold a talk in community, for this topic, exchange our ideas~

**Review by maroonstar — CHANGES_REQUESTED** [review]
That's a great idea! But this is not a task or  a course, put it in isses would be better I think. 😄

File: nft/nft与数字身份随想.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+通过这种设置，我们可以使用一个账号聚合自己所有地址的行为NFT，变相的就是把我们可以将所有的链上行为聚合到一起。
+
+这么做的好处，是项目方可以通过查询NFT的方式，识别一个账号所有的历史行为，从而对这个账号形成一个立体判断。
+
+**可以用于空投符合条件的查询，可以用于白名单设置的查询，可以进行简单的身份符合度验证，也可以实现更加复杂的用户画像和用户标签的建立。进一步也可以做一个大胆的尝试，实现无第三方中介的链上信用借贷。**
+
+为了避免可能的身份泄露，全部的行为NFT都是基于用户自我授权的，也就是用户可以选择绑定哪些账号的行为到一个NFT中，也可以隐藏不像公开的地址。这样用户可以很好的控制隐私。
+
+信用借贷由于无抵押，所以要根据所有的历时行为计算借款人的偿付能力。同时，为了做最后一道保障，也就是借款人跑路的情况下，如何去追偿。
```
Comment (liberhe) [inline nft/nft与数字身份随想.md:74]: "历时行为"

File: nft/nft与数字身份随想.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+我们可以做以下几个尝试：
+- 1、借贷设置为有限期，比如3-6个月
+- 2、在链上做借款人的真实身份信息校验，但只有身份提供方进行链上的加密私密保存，不像任何人公开
+- 3、只有当借款人未偿付时，才会触发系统调用真实身份信息进行追偿
+- 4、如果按期还款，真实身份信息将被销毁，过程中只有借款人自己接触过真实信息，因此不会有任何泄露
+- 5、未偿付行为可以设立保险基金之类的，通过保险质押奖励来补偿借贷空缺
+
+以上仅是利用链上数字身份开展的一个信用借贷的假想，实际还会有更多其他的玩法可以基于数字身份开展。
+
+这些是我一个初步的产品设计思考，有感兴趣的小伙伴可以联系我，大家一起探讨，没准可以做成一个项目。或者有技术实力的团队也可以去尝试下是否可行~
```
Comment (liberhe) [inline nft/nft与数字身份随想.md:85]: what you want to do is to implant Web3 identity? Is it safe to say so?

**Review by maroonstar — APPROVED** [review]

**Review by maroonstar — APPROVED** [review]

**Review by liberhe — APPROVED** [review]

**Review by maroonstar — APPROVED** [review]

**Pull request merged**